shared/src/main/scala/amf/core/model/domain/Linkable.scala

```diff
@@ -40,7 +42,7 @@ trait Linkable extends AmfObject { this: DomainElement with Linkable =>
     copied
       .withId(s"${copied.id}/link-$hash")
       .withLinkTarget(this)
-      .withLinkLabel(label)
+      .set(LinkableElementModel.Label, AmfScalar(label, annotations), Annotations.inferred())
```

Comment:
can't withLinkLabel accept annotations or add the default ones instead of using the "set" primitive?

Reply:
The set runs over the "copied" object, not over the link label field.
Annotations are being added over the new object.

Reply:
lgtm